plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -697,6 +699,44 @@ public void attachContainer(final AttachContainerParams params, MessageProcessor
     }
 
     /**
+     * Get stdout and stderr logs from container.
+     *
+     * @param containerLogsProcessor
+     *         output for container logs
+     * @throws ContainerNotFoundException
+     *         when container not found by docker (docker api returns 404)
+     * @throws IOException
+     *         when a problem occurs with docker api calls
+     */
+    public void getContainerLogs(final GetContainerLogsParams params, MessageProcessor<LogMessage> containerLogsProcessor)
```

Comment:
Please add tests for that method. Here is an example https://github.com/eclipse/che/compare/master...dockerNetworkAPI#diff-7f46e9ccf9514379f197f5dac7063f39R1388